packages/fuels-test-helpers/Cargo.toml

```diff
@@ -35,3 +35,4 @@ which = { workspace = true, default-features = false }
 default = ["fuels-accounts", "std"]
 std = ["fuels-accounts?/std", "fuels-core/std"]
 fuel-core-lib = ["fuel-core"]
+test-against-live-node = []
```

Comment:
I'd rather we don't introduce a feature if we don't have to. Since the test code compiles always (unlike the case with full sway type paths tests) I'd rather we make the decision to target the test net based on an env variable:
```rust
if env!(TARGET_TEST_NET) ... 
```

This way our internal testing approach doesn't leak into user features.